org.postgresql.util.PSQLException: FEHLER: Relation »direct_output« existiert bereits
Quitting on error...
